.vscode/settings.json

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
Get this VS stuff outta here! :D And add `.vscode` to gitignore will ya? :smile:

Reply:
Nahhh I think I'll keep it 😄